lib/libutils/ext/include/util.h

```diff
@@ -188,7 +188,34 @@ static inline void reg_pair_from_64(uint64_t val, uint32_t *reg0,
 	*reg1 = low32_from_64(val);
 }
 
-/* Get and set bit fields  */
+/*
+ * Functions to get and set bit fields in a 32/64-bit register.
+ *
+ * These helper functions allow setting and extracting specific bits in
+ * a bitfield @reg according to a given mask @mask. The mask
+ * specifies which bits in the register should be updated or extracted.
```

Comment:
s/register/input 32bit or 64bit data/

Reply:
OK I've updated the text to remove register